### ISSUE TITLE
syntaxerror:invalid syntax

### ISSUE THREAD

**Issue by wangxinzhe123** [MatNLP/HiCLRE#2]
Incomplete code，
in HiCLRE/hiclre/framework/data_loader.py
，Line 221 

 seqs[i] = torch.cat(seqs[i],

**Comment by fromdongyang** [comment]
seqs[i] = torch.cat(seqs[i],dim=0)
